packages/block-library/src/separator/style.scss

```diff
@@ -8,7 +8,8 @@
 
 	// Dots style
 	&.is-style-dots {
-		background: none; // Override any background themes often set on the hr tag for this style.
+		background: none !important; // Override any background themes often set on the hr tag for this style.
```

Comment:
Why is the "important" necessary? Should we move the comments before the rule now that it's a multiline comment?